statsd/statsd.go

```diff
@@ -185,6 +185,13 @@ func (c *Client) Set(name string, value string, tags []string, rate float64) err
 	return c.send(name, stat, tags, rate)
 }
 
+// Sends timing information in milliseconds. Flushed by statsd with percentiles, mean and other info
```

Comment:
golint will complain about this because Go documentation for functions/methods start off like `<FunctionName> ...`